utils/rjson.hh

```diff
@@ -416,6 +416,12 @@ public:
     }
 };
 
+inline bool is_leaf(const rjson::value& value) {
```

Comment:
Do you think someone will want to use this new rjson::is_leaf from other places? If not, I would move this tiny implementation detail to rjson.cc.

Reply:
I believe it's useful as a standalone function.